tests/e2e/etcd_config_test.go

```diff
@@ -449,4 +449,6 @@ func TestEtcdTLSVersion(t *testing.T) {
 	assert.NoError(t, e2e.WaitReadyExpectProc(context.TODO(), proc, e2e.EtcdServerReadyLines), "did not receive expected output from etcd process")
 	assert.NoError(t, proc.Stop())
 
+	proc.Wait() // ensure the port has been released
+	proc.Close()
```

Comment:
```suggestion
	assert.NoError(t, proc.Close())
```

Reply:
@ahrtr I think we should ignore error here because, when etcd isn't pid 1 process, it will clear signal handler and kill itself by the sigterm, after teardown. It always returns error here in our test cases.

https://github.com/etcd-io/etcd/blob/bc0f724ae42e876d1df6501c6c12c00e8d484e9b/pkg/osutil/interrupt_unix.go#L72-L78

Reply:
I see lots of such test code, so we just exclude `unexpected exit code`? 

```
	err = ep.proc.Close()
	if err != nil && !strings.Contains(err.Error(), "unexpected exit code") {
		return err
	}
```

Reply:
Since the purpose of this case is to test tls flag, so I am OK to keep it as it's

Reply:
Yes. We just need to ensure that server has been terminated and all the resources are reaped by kernel. So that it won't impact the following test.